protocol/message.go

```diff
@@ -8,6 +8,9 @@ import (
 // of messages exchanged by clients and servers.
 const (
 	RegistrationType = iota
+	KeyLookupType
+	KeyLookupInEpochType
+	ConsistencyCheckType
```

Comment:
I think `MonitoringType` might be a better name for `ConsistencyCheckType` since that's really the only time a client should send this type of request.